### ISSUE TITLE
Miscellaneous section

### ISSUE THREAD

**Issue by seanfisk** [seanfisk/mastering-eos#29]
- [x] Ctrl-Alt-Backspace to kill the X server
- [x] directory navigation (`pushd`, `popd`, `dirs`)
- [x] uptime
- [x] user interaction (`w`, `last`, `users`, `write`, `mesg`)
- [x] `xdg-open`
- [x] clipboard interaction (`xclip`)
- [ ] downloading files (`wget`, `curl`, and `aria2c`)
- [ ] running Make in parallel
- [ ] Job management
  - [ ] `pgrep`, `pkill`, and `killall`
  - [ ] `ps`
  - [ ] `jobs`
  - [ ] backgrounding commands (`&`), `fg`, and `bg`
- [ ] taking screenshots
- [ ] printing syntax-highlighted code (check out [enscript](https://www.gnu.org/software/enscript/) and a [blog post on it](http://blog.vinceliu.com/2008/06/printing-syntax-highlighted-source-code.html))

**EDIT 2015-02-16:** tmux should have its own section. I've removed it from this list and created a separate issue for it.
**EDIT 2015-03-12:** Added screenshots and code printing.
